larq/quantizers.py

```diff
@@ -50,12 +50,7 @@
 
 import tensorflow as tf
 from larq import utils, math
-
-try:
-    from tensorflow import is_tensor
-except:  # pragma: no cover
-    # For compatibility with TensorFlow 1.13
-    from tensorflow.python.framework.tensor_util import is_tensor
+from tensorflow import is_tensor
```

Comment:
This import is only there due to TF 1.13 compatibility. I'd prefer to use `tf.is_tensor` instead.

Reply:
Yeah that's cleaner; changed in 310ace2.